app/views/teams/show.html.erb

```diff
@@ -18,41 +17,80 @@
+                                :class => 'btn btn-default btn-xs',
+                                :disabled => !(can? :delete_ownership, @team) %>
+                <% else %>
+                    <%= link_to t('.assign_ownership', :default => t("helpers.links.assign_ownership")),
+                                assign_ownership_team_path(@team, :team_member => member.id),
+                                :method => 'post',
+                                :class => 'btn btn-default btn-xs',
+                                :disabled => !(can? :assign_ownership, @team)%>
+                <% end %>
+                <%= link_to t('.delete_membership', :default => t("helpers.links.delete_membership")),
+                            delete_membership_team_path(@team, :team_member => member.id),
+                            :method => 'post',
+                            :class => 'btn btn-xs btn-danger',
+                            :disabled => !(can? :delete_membership, @team, member.id)%>
+            <% end %>
+          </td>
+        </tr>
     <% end %>
-  </tbody>
-</table>
+    <tr>
+      <td>
+        <%= t('.apply_action', :default => t("helpers.teams.apply_action")) %>
+      </td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td>
+        <%= link_to t('.email', :default => t("helpers.links.email")),
+                    team_path(@team),
+                    :method => 'get',
+                    :class => 'btn btn-default btn-xs' %>
+
+        <%= submit_tag t('.assign_ownership', :default => t("helpers.links.assign_ownership")), name: "assign_ownership" %>
+        <%= submit_tag t('.delete_ownership', :default => t("helpers.links.delete_ownership")), name: "delete_ownership" %>
+        <%= submit_tag t('.delete_membership', :default => t("helpers.links.delete_membership")), name: "delete_membership" %>
+      </td>
+    </tr>
+    </tbody>
+    <thead>
+    <tr>
+      <th><%= t '.select', :default => t("helpers.select.prompt") %></th>
+      <th><%= model_class.human_attribute_name(:id) %></th>
+      <th><%= model_class.human_attribute_name(:first_name) %></th>
+      <th><%= model_class.human_attribute_name(:last_name) %></th>
+      <th><%= t '.actions', :default => t("helpers.actions") %></th>
+    </tr>
+    </thead>
+  </table>
+<% end %>
 
 <%# app/helpers/button_helper.rb %>
 <%= back_btn %>
-<% if can? :edit, @team %>
-    <%= link_to t('.edit', :default => t("helpers.links.edit")),
-                edit_team_path(@team), :class => 'btn btn-default' %>
-<% end %>
-<% if can? :destroy, @team %>
-    <%= link_to t('.destroy', :default => t("helpers.links.destroy")),
-                team_path(@team),
-                :method => 'delete',
-                :data => {:confirm => t('.confirm', :default => t("helpers.links.confirm"))},
-                :class => 'btn btn-danger' %>
-<% end %>
\ No newline at end of file
+<%= link_to t('.edit', :default => t("helpers.links.edit")),
```

Comment:
Any reason why you removed the can? checks here? It’s important that e.g. non-team owners can’t delete teams (and thus don’t get the UI button displayed)